app/js/components/manageApps/Addon.jsx

```diff
@@ -277,29 +285,30 @@ class Addon extends Component {
               <button
                 type="button"
                 className="btn btn-primary module-control"
-                onClick={(event) => this.handleAction(event, app.uuid, "stop")}
+                onClick={(event) => this.handleAddonAction(event, "stop")}
```

Comment:
You should use either `event` or `e` throughout the code so as not to confuse a beginner that would be reading the code sometimes.

Reply:
Have used event